### PR TITLE
Add examples/models/files/graphs.py example

### PR DESCRIPTION
**Pull request by canavandl**
needs to be merged after #6686 

looks like (each with different interactions):
screen shot 2017-07-31 at 5 25 01 pm

### CODE REVIEW

File: examples/examples.yaml [hunk 1]
```diff
@@ -1,6 +1,7 @@
 - path: "models/file"
   slow: ["choropleth.py"]
   no_js: ["maps_cities.py", "maps.py", "trail.py"]
+  no_diff: ["graphs.py"]
```
Comment (canavandl) [inline examples/examples.yaml]: it'll be a no_diff example because the networkx layout methods aren't all deterministic

Comment (mattpap) [inline examples/examples.yaml]: Why is that? Is there anything we can do other than simply silencing this example? If there is a way, it should be preferred over no_diff or no_js.

Comment (canavandl) [inline examples/examples.yaml]: You're right that doing no_diff reduces its test value. I'll modify the example to use the two deterministic layout algos and remove the no_diff flag.

Comment (canavandl) [inline examples/examples.yaml]: Looking at it, the only deterministic layout function is nx.circular_layout. Having only that layout reduces the example value too much in my opinion. Would you be opposed to an additional screenshot test of a circular layout example?

**Pull request merged**